cinema/serializers.py

```diff
@@ -68,13 +84,62 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available"
         )
 
 
+class TicketSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ticket
+        fields = ("id", "row", "seat", "movie_session")
+
+    def validate(self, attrs: dict) -> None:
+        Ticket.validate_row_and_seat(
+            row=attrs["row"],
+            seat=attrs["seat"],
+            cinema_hall=attrs["movie_session"].cinema_hall,
+            error_to_raise=serializers.ValidationError
+        )
+
```

Comment:
check if the POST method works for the order. I have doubts. Since validation should return attr

Reply:
Yes, it works. Why should it return attr? Validation should raise an errror if some data is not valid.

Reply:
validate should return data if they valid or raise error. 
i think so*

Reply:
I checked it again. You're right. It should return data here.

Reply:
Here would make sense to call super() and return result from there